### DISCUSSION TITLE
how to remove scroll bar for grid in vaadin 24

### DISCUSSION THREAD

**Discussion by HuyQuocPhamlk** [community/community#55195] [Programming Help]
### Select Topic Area

Question

### Body

how to remove scroll bar for grid in vaadin 24
![anhcode1](https://github.com/community/community/assets/97649721/25db1d80-96aa-4a8e-9511-b224f7610c47)
![anhkq](https://github.com/community/community/assets/97649721/f2cca04c-042f-4e3d-91a0-463dbe0c8914)
![anhcode](https://github.com/community/community/assets/97649721/a5707e51-6e1b-417b-8979-294121e07537)

**Comment by rainbow1016** [comment]
very good question!
